api/src/main.rs

```diff
@@ -1,10 +1,9 @@
 use config::Configuration;
+use dotenv::dotenv;
+use envconfig::Envconfig;
 use once_cell::sync::OnceCell;
 use std::process;
 use warp::Filter;
-extern crate log;
```

Comment:
I have this to make it available in the whole crate. This might not be needed for `diesel`, but for `log` it could be quite helpful.

Reply:
For me all the libraries are still available, even without the `extern crate`-statement